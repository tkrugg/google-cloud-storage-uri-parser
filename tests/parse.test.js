const {
    parse
} = require('../');
describe('parse', () => {
    const tests = ["gs://mybucket/myfile.json", "gs://my-bucket_with/myfolder/myfile.json", ]
    for (const test of tests) {
        it(`should produce the right bucket and file names for ${test}`, () => {
            expect(parse(test)).toMatchSnapshot()
        })
    }
});