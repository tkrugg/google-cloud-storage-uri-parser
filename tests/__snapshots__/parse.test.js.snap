// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parse should produce the right bucket and file names for gs://my-bucket_with/myfolder/myfile.json 1`] = `
Object {
  "bucket": "my-bucket_with",
  "file": "myfolder/myfile.json",
  "uri": "gs://my-bucket_with/myfolder/myfile.json",
}
`;

exports[`parse should produce the right bucket and file names for gs://mybucket/myfile.json 1`] = `
Object {
  "bucket": "mybucket",
  "file": "myfile.json",
  "uri": "gs://mybucket/myfile.json",
}
`;
